src/Components/ArticleCards.jsx

```diff
@@ -0,0 +1,17 @@
+import './articleCards.css'
+import { calculateYears } from '../../utils/utils'
+
+export default function ArticleCards ({articles}) {
+    return  <ul>
+    {articles.map((article) => {
+         const yearsSincePosted = calculateYears(article.created_at)
+        return (<li key = {article.article_id} className="article-card">
```

Comment:
You could if you wanted make another component to return an individual article card and return it from the map